lesson2/fib/fib.go

```diff
@@ -0,0 +1,17 @@
+package fib
+
+import "fmt"
+
+func OutFibanacciLine(x int) {
```

Comment:
One of the best practices at coding is SOLID (google about it). First S mean "single responsibility": one function for printing numbers and second one is for generating them. You code works, but let us to do it better!

Reply:
if divided into two functions, then I need an additional  array. At the time of writing the code, I did not know arrays)